tests/test_octodns_record.py

```diff
@@ -1512,6 +1512,9 @@ def test_mx_value(self):
         self.assertTrue(c >= c)
         self.assertTrue(c <= c)
 
+        self.assertEqual(a.__hash__(), a.__hash__())
+        self.assertNotEqual(a.__hash__(), b.__hash__())
+
```

Comment:
what's up here? 🤔

Reply:
This is getting some reasonable coverage for `MxValue.__hash__()`. Apparently RackspaceProvider was somehow hitting those code paths and nothing else was. I have a note to look into what's up there, but whatever it is it seemed out of scope for removing the provider.

Reply:
Just a suspicion:
```
    def test_apply_create_MX(self):
        class TestData(object):
            OtherRecords = [
                {
                    "subdomain": '',
                    "data": {
                        'type': 'MX',
                        'ttl': 300,
                        'value': {
                            'value': 'mail1.example.com.',
                            'priority': 1,
                        }
                    }
                },
```
(in `test_octodns_provider_rackspace.py`, l. 342 ff.) hits

```
        # UNTIL 1.0 remove value fallback
        try:
            exchange = value['exchange']
        except KeyError:
            exchange = value['value']
        self.exchange = exchange.lower()
```
(in `record/__init__.py`, l. 1111 ff.).